store/conf/web.xml.production

```diff
@@ -222,7 +222,8 @@
       <location>/opt/zimbra/data/tmp</location>
       <max-file-size>%%zimbraFileUploadMaxSize%%</max-file-size>
       <max-request-size>%%zimbraMailContentMaxSize%%</max-request-size>
-      <file-size-threshold>%%zimbraFileUploadMaxSize%%</file-size-threshold>
+      <!-- The file size in bytes after which the file will be temporarily stored on disk -->
```

Comment:
No, it will not happen we have long supported value for them.

Reply:
ok should we make this value configurable? or is it not required?

Reply:
It is just a threshold value and by default is 0 but we are taking 1MB here to save some disk I/O and It will not require to change so I think it's not required. 
what is your opinion?

Reply:
I am ok if we feel this should not be configurable, I was just curious.